Keithley_2700Sup/devKeithley_2700.proto

```diff
@@ -0,0 +1,174 @@
+##################################################
+#
+# Protocol file for Keithley 2700
+#
+##################################################
+
+Terminator = '\r\n';
```

Comment:
We discovered that the terminator on the 2400 is configurable. Is the same true of the 2700? If so, it should probably be allowed to vary here too (see KHLY2400)

Reply:
Added lines to st.common to allow for configurable terminators. From manual:
```
The Model 2700 can be configured to terminate each program message that it transmits to
the controller with any of the following combinations of <CR> and <LF>.
<CR> Carriage return
<CR+LF> Carriage return and line feed
<LF> Line feed
<LF+CR> Line feed and carriage return
```